src/misc/unicode.cpp

```diff
@@ -685,24 +699,25 @@ static bool utf8_to_wide(const std::string& str_in, std::vector<uint16_t>& str_o
 				code_point = code_point + byte_2 -
 				             DecodeThresholdNonAscii;
 			} else {
-				status = false; // code point encoding too short
+				// code point encoding too short
+				warn_decode_problem(i);
 			}
 		} else if (byte_1 < DecodeThresholdNonAscii) {
 			// 1-byte code point, ASCII compatible
 			code_point = byte_1;
 		} else {
-			status = false; // not UTF8 encoding
+			warn_decode_problem(i); // not UTF8 encoding
 		}
 
 		str_out.push_back(static_cast<uint16_t>(code_point));
 	}
 
-	return status;
+	return str_out;
 }
 
-static void wide_to_utf8(const std::vector<uint16_t>& str_in, std::string& str_out)
+static std::string wide_to_utf8(const wide_string& str_in)
```

Comment:
Just `str` instead of `str_in` (all `const` args are inputs, that's obvious).

Reply:
OK, changed everywhere.